aws-analytics-pinpoint/src/androidTest/java/com/amplifyframework/analytics/pinpoint/AnalyticsPinpointInstrumentedTest.java

```diff
@@ -191,6 +200,43 @@ public void testUnregisterGlobalProperties() throws AnalyticsException {
         assertFalse(analyticsClient.getAllEvents().get(0).has("attributes"));
     }
 
+    @Test
+    public void testIdentifyUser() {
+        // Create a location object
+        AnalyticsUserProfile.Location location = new AnalyticsUserProfile.Location(SEATTLE_LATITUDE,
+                SEATTLE_LONGITUDE,
+                "98122",
+                "Seattle",
+                "WA",
+                "USA");
```

Comment:
Instead of splitting the magic data between class-level constants, and inline literals, I would just keep it all locally defined. In other words, the instance of `AnalyticsUserProfile.Location` _itself_ is magic data, not just its constituent fields:

```
@SuppressWarning("checkstye:MagicNumber") // Seattle's location is magic. Snow-capped volcanos, Orcas swimming in the Puget Sound, and the smell of fresh roasted coffee wafting on the pier...
UserProfile.Location locationOfSeattle = UserProfile.Location.builder()
    .latitute(47.6154086)
    .longitude(-122.3349685)
    .zipcode("98122")
    .city("Seattle")
    .locality("WA") // locality? TODO: what is the generic term for this, that accounts for states/provinces?
    .country("USA")
    .build();
```